src/newcharacter.cpp

```diff
@@ -3489,6 +3501,27 @@ static void draw_gender( ui_adaptor &ui, const catacurses::window &w_gender,
     wnoutrefresh( w_gender );
 }
 
+static void draw_outfit( ui_adaptor &ui, const catacurses::window &w_outfit,
+                         const avatar &you, const bool highlight )
+{
+    const point male_pos( 1 + utf8_width( _( "Outfit:" ) ), 1 );
+    const point female_pos = male_pos + point( 2 + utf8_width( _( "Male" ) ), 1 );
```

Comment:
```suggestion
    const point male_pos( 1 + utf8_width( _( "Outfit:" ) ), 0 );
    const point female_pos = male_pos + point( 2 + utf8_width( _( "Male" ) ), 0 );
```

The window has a height of 1, so the y pos needs to be 0 to draw it inside the window bounds here.